### DISCUSSION TITLE
TPI MACOWINS MANEJO DE ERRORES

### DISCUSSION THREAD

**Discussion by MateoSilva626** [dds-jv/foro#346] [Consultas generales] [answered]
Les hago una consulta, hay que tener en cuenta el manejo de errores para este tp? No sé hasta qué profundidad hay que desarrollarlo. Desde ya, muchas gracias!

**Comment by gastonprieto** [comment, marked as answer]
buenas @MateoSilva626 

Creo que no habia ningún caso particular de errores de macowins, asi que no haria falta, pero si se te ocurre alguno podes agregarlo.

**Reply by MateoSilva626** [reply]
Genial, muchas gracias!
